internal/logging/zerolog.go

```diff
@@ -57,13 +56,22 @@ func decorate(l zerolog.Logger) zerolog.Logger {
 // InitializeStdout initializes logging to standard output with human-friendly output.
 // It is used before CloudWatch logging output is initialized, or in unit and integration tests.
 func InitializeStdout() {
+	var noColor bool
+
+	// Do not use colors when running in clowder
+	if config.InClowder() {
```

Comment:
```suggestion
	if config.InEphemeralClowder() {
```

Reply:
This is not a typo, I introduced the function `InEphemeralClowder` but the intention was not to use it.

Reply:
Ah, oki ;) i did not read well then :)